adb_graphics/datahandler/gribdata.py

```diff
@@ -613,10 +613,11 @@ def grid_info(self):
             attrs = ['Lov']
             grid_info['projection'] = 'stere'
             grid_info['lat_0'] = 90
-            grid_info['lat_ts'] = 90
```

Comment:
The documentation on lat_ts states: "latitude of true scale. Optional for stereographic, cylindrical equal area and mercator projections. default is lat_0 for stereographic projection. default is 0 for mercator and cylindrical equal area projections."  So it defaults to lat_0, which is 90, and thus not needed.  It may be that it will need to be set for the Southern Hemisphere plot, when that is available.